### PR TITLE
Add Create and edit geometries sample (QML only)

### PR DESCRIPTION
**Pull request by AndrewBladon**
# Description

The sample design for the new `Create and edit geometries` sample is in the later stages of the sample design review process. Although the design has not yet been finalised, the design and this version of the sample have been seen by the Samples Design Council and several changes have been made following their input. It is not anticipated that any significant (e.g. behavioural) changes will be requested from the Samples Design Council.

This PR adds the QML version of the `Create and edit geometries` sample, which was used to assist in the design process. An issue will be added to the Qt repository to capture the remaining work required to complete the addition of the sample (e.g. creation of the C++ version of the sample and addition of the sample to sample viewer).

Outstanding tasks to be completed before merging this PR:

- [ ] Add README.md after approval from the Samples Design Council.
- [ ] Update metadata.json.
- [ ] Once This PR has been approved, the code in `CreateAndEditGeometries.qml` should be added to the sample design as the `implementation-details.md` file.

## Type of change

- [ ] Bug fix
- [x] New sample implementation
- [ ] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [x] Windows
- [ ] Android
- [ ] Linux
- [ ] macOS
- [ ] iOS

## Checklist

- [ ] Runs and compiles on all active platforms as a standalone sample
- [ ] Runs and compiles in the sample viewer(s)
- [x] Branch is up to date with the latest main/v.next
- [ ] All merge conflicts have been resolved
- [x] Self-review of changes
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files
- [x] Code is commented with correct formatting (CTRL+i)
- [x] All variable and method names are camel case
- [x] There is no leftover commented code
- [x] Screenshots are correct size and display in description tab (500px by 500px, platform agnostic)
- [ ] If adding a new sample, it is added to the sample viewer
- [ ] Cherry-picked to Main branch (if applicable)

### CODE REVIEW

**Comment by tanneryould** [comment]
Looks good so far! Thanks for doing this!

File: ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateAndEditGeometries/CreateAndEditGeometries.qml [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+                        }
+
+        onIdentifyGraphicsOverlayStatusChanged: {
+            if (identifyGraphicsOverlayStatus !== Enums.TaskStatusCompleted)
+                return;
+
+            if (mapView.identifyGraphicsOverlayResult.graphics.length === 0)
+                return; // No graphic at location of tap.
+            else if (mapView.identifyGraphicsOverlayResult.graphics.length > 1) {
+                console.log(qsTr("Multiple graphics identifed - cannot identify the intended graphic."));
```
[the remark this answers is not in the document]
Comment (AndrewBladon) [reply ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateAndEditGeometries/CreateAndEditGeometries.qml]: Change made in https://github.com/Esri/arcgis-maps-sdk-samples-qt/pull/1594/commits/683a966ac2631527d78572d1ec333e4bd0489f35.

File: ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateAndEditGeometries/CreateAndEditGeometries.qml [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+
+    function createLineAndAddToGraphics(lineCoords) {
+        let polylineBuilder = ArcGISRuntimeEnvironment.createObject("PolylineBuilder", {spatialReference: Factory.SpatialReference.createWgs84()});
+        lineCoords.forEach( (coordinate) => {polylineBuilder.addPointXY(coordinate[0], coordinate[1])});
+        const polylineGeometry = polylineBuilder.geometry;
+        const polylineGraphic = ArcGISRuntimeEnvironment.createObject("Graphic", {geometry: polylineGeometry, symbol: polylineSymbol});
+        geometriesOverlay.graphics.append(polylineGraphic);
+    }
+
+    function createPolygonAndAddToGraphics(polygonCoords) {
```
[the remark this answers is not in the document]
Comment (AndrewBladon) [reply ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateAndEditGeometries/CreateAndEditGeometries.qml]: Change made in https://github.com/Esri/arcgis-maps-sdk-samples-qt/pull/1594/commits/f77d45d0024ce4072ddc70f49b19158f678d8d1f.

File: ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateAndEditGeometries/README.metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "category": "TODO",
```
[the remark this answers is not in the document]
Comment (AndrewBladon) [reply ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateAndEditGeometries/README.metadata.json]: `metadata.json` was updated using the python script. These changes were committed in https://github.com/Esri/arcgis-maps-sdk-samples-qt/pull/1594/commits/e40c87b06a85be542bf67d4084f8f2e63899642b.

**Comment by AndrewBladon** [comment]
Thanks @ldanzinger and @tanneryould for your comments. I've made changes and now believe the PR has all of the components required to merge (e.g. completed `Readme.metadata.json` file). Please could you give these latest changes a review?

**Review by ldanzinger — APPROVED** [review]
LGTM..

**NOTE** - probably best to hold off merging for now until we go live (Wednesday). That will give us more time to get the C++ version done and test it properly before it ends up in the main branch

**Comment by AndrewBladon** [comment]
>probably best to hold off merging for now until we go live (Wednesday). That will give us more time to get the C++ version done and test it properly before it ends up in the main branch
Thanks @ldanzinger, I'll wait and merge later in the week.

**Pull request merged**